net/foolsm/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2010-2011 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=foolsm
+PKG_VERSION:=1.0.10
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://lsm.foobar.fi/download
+PKG_HASH:=33210209ca38b3bfef1a9180f765266a134fc811dea8bc06450a3bd48d1d083e
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/foolsm
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+ssmtp
```

Comment:
I don't think you actually depend on ssmtp at all, just that the script you include as an example that could be run uses ssmtp, is that correct?

Reply:
Yes, script that will be in a default installation, assuming the user doesn't do anything to update/override [requires ssmtp](https://github.com/openwrt/packages/pull/5273/files?diff=unified#diff-ef23fafb1de13a01f19d1247052766daR25).